pkg/operator/operator.go

```diff
@@ -58,6 +58,23 @@ func ValidateOperatorInstallStatus(cs *clients.Clients, rnames config.ResourceNa
 	log.Printf("Operator is up\n")
 }
 
+func ValidateChainsConfigMaps(cs *clients.Clients, rnames config.ResourceNames) {
+	AssertConfigMapPresent(cs, rnames.TargetNamespace, "chains-config")
+	AssertConfigMapPresent(cs, rnames.TargetNamespace, "chains-info")
+}
+
+func ValidateChainsDeployments(cs *clients.Clients, rnames config.ResourceNames) {
+	EnsureTektonChainExists(cs.TektonChain(), rnames)
```

Comment:
Shouldn't we verify if there are no errors before proceeding to next step?

Reply:
I will add error handling here. The same problem is above with other `Ensure***Exists` calls